pkg/cmd/version.go

```diff
@@ -0,0 +1,37 @@
+// Copyright © 2019 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/cli"
+)
+
+const version = "0.1.0"
```

Comment:
We can do the same as [`knative/client`](https://github.com/knative/client/blob/master/pkg/kn/commands/version.go#L23), and using `ldflags` — this will help us detect "dev" version from released version *and* not change anything in this file when releasing.

Reply:
+1 to what @vdemeester said. 

we should be able to use `go build -ldflags "-X $PACKAGE_PATH/version.ClientVersion=$(git describe)"`

Reply:
Make sense.